index.html

```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Josh Lee</title>
+
+  <!-- Leave these links here...
+  But this should be in a comment... -->
+  <link href="https://fonts.googleapis.com/css?family=Cormorant|Poppins:300,400" rel="stylesheet"/>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link type="text/css" rel="stylesheet" href="styles/style.css"/>
+</head>
+<body>
+  <div class="app">
+    <nav class="app__nav">
+      <div class="container">
+        <div class="top-nav">
+          <h1 class="top-nav__left">Josh Lee</h1>
+
+          <div class="top-nav__left">
+            <ul class="nav-list">
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#welcome">Welcome</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#about">About</a>
+              </li>
+              <li class="nav-list__item">
+                  <a class="nav-list__link" href="#quotes">Quotes</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#photos">Photos</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <div class="app__content">
+      <section class="section--hero" id="welcome">
+        <div class="container">
+          <div class="hero">
+            <div class="hero__item">
+              <h2 class="hero__text">I've never lived anywhere for more than 3 years!</h2>
+            </div>
+            <div class="hero__item">
+              <img src="http://www.bobbyberberyan.com/wp-content/uploads/2012/03/HTML5CSS3Logos.svg" alt="" class="zapp__img">
+            </div>
+          </div>
+        </div>
+      </area-of-control>
+
+      <div class="section">
+        <div class="container">
+          <h2 class="section__heading">Look where I've been!</h2>
+
+          <div class="grid">
+            <div class="grid__item">
+              <div class="card">
+                <div class="media">
+                  <div class="circle-icon">
```

Comment:
Check the class of this div here. The original class name was `twitter__first` and it looks like you replaced `twitter` with `media` so this could be a big cause to why you're getting square images.